main.go

```diff
@@ -52,6 +53,24 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+func LookupEnvOrString(key string, defaultVal string) string {
```

Comment:
Could you please make this func private?

Reply:
sure